solutions/usaco-894.mdx

```diff
@@ -117,6 +117,29 @@ public class GrassPlanting {
 
 </JavaSection>
 
+<PySection>
+
+```py
+fin = open("planting.in", 'r')
+fout = open('planting.out', 'w')
+inputs = [i for i in fin.read().splitlines()]
+
+n = int(inputs[0])
+deg = [1 for i in range(n+1)]
+maxDeg = 0
+
+# Tracking max degree
+for i in inputs[1:]:
+	u,v = [int(j) for j in i.split(" ")]
+	deg[u] += 1 
+	deg[v] += 1 
+	maxDeg = max(maxDeg, deg[u], deg[v]);
+
+print(maxDeg, file=fout)
```

Comment:
`fout` is redundant, just directly put it in here

Reply:
like
`print(max_deg)`
`print(max_deg, fout)`
`print(max_deg, file)`?
I tried those 3 and it doesn't seem to work

Reply:
`print(max_deg, open("filename.out", "w"))`